src/test/java/org/folio/rest/impl/RtacResourceImplTest.java

```diff
@@ -396,7 +396,7 @@ public final void testGetRtacByIdVolumeFormatting() {
     final JsonObject expectedJson = new JsonObject(
         readMockFile("RTACResourceImpl/success_rtac_response_volume_formatting.json"));
 
-    final int expectedSize = 7;
+    final int expectedSize = 8;
     assertEquals(expectedSize, json.getJsonArray("holdings").size());
     for (int i = 0; i < expectedSize; i++) {
       final JsonObject jo = json.getJsonArray("holdings").getJsonObject(i);
```

Comment:
Not being picky but this "jo" object could be renamed to something more meaningful, such as "holdingsJo"

Reply:
Do you think this clean up should be done with this story or with a tech debt story? I have no preference one way or the other... except maybe to minimize changes.

Reply:
Let's create a tech debt story, but I have a feeling that we will not have time to work on tech debt stories. It took me a second look to follow the code correctly so it's not too bad, not 5 times :)

Reply:
Done: [MODRTAC-27](https://issues.folio.org/browse/MODRTAC-27)